pages/cards/cards.html

```diff
@@ -0,0 +1,166 @@
+      <section class="section">
+        <article class="section-content underline">
+          <h2>Card</h2>
+          <p class="small-text">Cards are great for organizing and enhancing a website's user interface. Each card is uniquely designed to make it easy for users to read the card content. </p>
+        </article>
+        <article class="section-content underline">
+          <h2>Basic Card with Shadow</h2>
+          <p class="small-text">Applying shadows to your card to show that they are on different plane from the rest of the UI.</p>
+          <div class="example-code">
+            <div class="example-container-flex">
+              <div class='card'>
+                <img id="card-image" src='../../assets/image1.jpg'>
+                <h3>Our changing planet</h3>
+                <p class="small-text">by Kurt Wagner</p>
+                <p class="small-text">Visit ten places on our planet that are undergoing the biggest changes today.</p>
+                <div class="card-footer">
+                  <button class="btn primary-btn">Read</button>
+                  <button class="btn btn-outline-primary">Bookmark</button>
+                  
+                </div>
+              </div>
+            </div>
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=1000&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--%2520use%2520card%2520class%2520on%2520the%2520parent%2520element%2520--%253E%250A%250A%253Cdiv%2520class%253D%27card%27%253E%250A%2520%2520%253Cimg%2520id%253D%2522card-image%2522%2520src%253D%27..%252F..%252Fassets%252Fimage1.jpg%27%253E%250A%2520%2520%253Ch3%253EOur%2520changing%2520planet%253C%252Fh3%253E%250A%2520%2520%253Cp%2520class%253D%2522small-text%2522%253Eby%2520Kurt%2520Wagner%253C%252Fp%253E%250A%2520%2520%253Cp%2520class%253D%2522small-text%2522%253EVisit%2520ten%2520places%2520on%2520our%2520planet%2520that%2520are%2520undergoing%2520the%2520biggest%2520changes%2520today.%253C%252Fp%253E%250A%2520%2520%253Cdiv%2520class%253D%2522card-footer%2522%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520primary-btn%2522%253ERead%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-outline-primary%2522%253EBookmark%253C%252Fbutton%253E%250A%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520"
```

Comment:
Is there a way you can minify this

Reply:
I will get back to you. 
As far as I know,  I don't think there is a way.